src/k8s-extension/HISTORY.rst

```diff
@@ -3,6 +3,12 @@
 Release History
 ===============
 
+1.5.4
+++++++++++++++++++
+* AAD related changes in dataprotection aks ext CLI
+* microsoft.azuremonitor.containers: Make containerlogv2 as default as true and remove region dependency for ARC
+* microsoft.workloadiam: Refactor subcommand invocation
```

Comment:
Could you add some tests for these PR changes?

Reply:
We have our own custom tests(CI checks) in our forked repo that create a kubernetes cluster and perform create, update, list, show and delete actions for each of the partner extensions.

[https://github.com/AzureArcForKubernetes/azure-cli-extensions/tree/k8s-extension/public/testing/test/extensions/public](https://github.com/AzureArcForKubernetes/azure-cli-extensions/tree/k8s-extension/public/testing/test/extensions/public)